src/components/Nav.vue

```diff
@@ -34,7 +34,9 @@
       id="navbarBasicExample"
       class="navbar-menu"
       :class="{ 'is-active': showNav }"
+      @click="showNav = false"
     >
+      <div class="burger-menu-overlay" @click="showNav = false" ></div>
```

Comment:
@SeddikKadi It looks like this div is spread over the app anytime. Could you make it so that it is only displayed when the menu is shown ?

Reply:
I don't want a useless div overlaid under everyting at any time, even if it's only for small creen. Sorry.
Can you make it so the `div class="burger-menu-overlay"` is only shown when the menu is open (the only moment when it is useful) ?
With something like: `v-if="showNav"` on the burger-menu-overlay itself should do the trick.

I have also a suggestion that goes a little further and does not take much LOC to implement. But really it is optional.
You could also put it right under the menu dropdown and the nav, and make it black an opacity 0.3 or similar (like a modal overlay) since what I understand from its behaviour is that it will catch any click and close the menu ? That would be a nice addition I think.

Reply:
i'll add the opacity overlay in another commit, still working on it ...

Reply:
Thanks a lot and sorry to waste your time with unprecise comments.